.env

```diff
@@ -30,5 +30,3 @@ ENTERPRISE_MARKETING_URL=''
 ENTERPRISE_MARKETING_UTM_SOURCE=''
 ENTERPRISE_MARKETING_UTM_CAMPAIGN=''
 ENTERPRISE_MARKETING_FOOTER_UTM_MEDIUM=''
-APP_ID=''
```

Comment:
This is because the load env variables commit set this empty and then our MFE_CONFIG_API_URL doesn't work.

https://github.com/nelc/frontend-app-gradebook/blob/81e155e463becaf84f3e86363cb9f2cdfa42b929/webpack.prod.config.js#L22-L25

I mean `/eox-nelp/...` because is changed by `''`

Reply:
Those .env values should be the default values  however those should be overridden by this ![image](https://github.com/nelc/frontend-app-gradebook/assets/36200299/377ebdf8-c64b-4bfe-85da-f2a6d24e8a01)

if that is not happening that means that the code that you showed has an error and we should fix that instead of modifying the env files.

I remember that one way to fix  the development variables was just to set the environment variables, that overrode a specific value of the env.development however I choose this approach  because this updates the whole file  but the intention never was to remove the priority that the environment variables have

Reply:
[b04ab36](https://github.com/nelc/frontend-app-gradebook/pull/3/commits/b04ab365954ef7408f3416a77ce09a36d7b7d904)

Reply:
Great :D